AbstractModels/AbstractUser.php

```diff
@@ -0,0 +1,35 @@
+<?php
+/**
+ * Model object generated by: Skipper (http://www.skipper18.com)
+ * Do not modify this file manually.
+ */
+
+namespace App\Models\AbstractModels;
+
+use Illuminate\Database\Eloquent\Model;
+
+abstract class AbstractUser extends Model
+{
+    /**
```

Comment:
Change all indentation from 2 spaces to 4 (2 is for js, 4 is PHP; see PSR-1, 2)

Reply:
done